CameraOverlay/OrchestratorClass.py

```diff
@@ -0,0 +1,78 @@
+"""Orchestrator Script That Controls The Camera System"""
+import json
+import argparse
+import time
+import paho.mqtt.client as mqtt
+import config
+
+def get_args(argv=None):
+    """Get arguments passed into Python script"""
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--host", type=str,
+                        default="192.168.100.100",
+                        help="ip address of the broker")
+    return parser.parse_args(argv)
+
+class Orchestrator():
+
+    def __init__(self, broker_ip , port = 1883):
+
+        self.BROKER_IP = broker_ip
```

Comment:
Unnecessary to have this in all caps. Variables with all caps usually denote a global variable. Since we have classes, we only have class or instance variables now. So no need to make this instance variable in all caps